components/gateway/Caddyfile

```diff
@@ -0,0 +1,169 @@
+		# order to pass the verification.
+		# Optional field.
+		# issuer_whitelist localhost
+ 
+		# AudienceWhitelist defines a list of audiences. A non-empty list turns
+		# on "aud verification": the "aud" claim must exist in the given JWT
+		# payload. The verification will pass as long as one of the "aud" values
+		# is on the whitelist.
+		# Optional field.
+		# audience_whitelist https://api.example.io https://learn.example.com
+
+
+		# UserClaims defines a list of names to find the ID of the authenticated
+		# user.
+		# By default, this config will be set to []string{"sub"}.
+		# If multiple names were given, we will use the first non-empty value of
+		# the key in the JWT payload as the ID of the authenticated user.
+		# i.e. The placeholder {http.auth.user.id} will be set to the ID.
+		# If no non-empty values found, leaves it unauthenticated.
+		# user_claims aud uid user_id username login
+
+		# MetaClaims defines a map to populate {http.auth.user.*} metadata
+		# placeholders. The key is the claim in the JWT payload, the value is
+		# the placeholder name. e.g. {"IsAdmin": "is_admin"} can populate
+		# {http.auth.user.is_admin} with the value of IsAdmin in the JWT payload
+		# if found, otherwise "".
+		# meta_claims "IsAdmin->is_admin"
+
+		# WithRefreshWindow specifies the interval between checks for refreshes.
+		#
+		# If not specified, the default value is 15 minutes.
+		#
+		# You generally do not want to make this value too small, as it can easily
+		# be considered a DoS attack, and there is no backoff mechanism for failed
+		# attempts.
+		refresh_window 10s
+
+		# WithMinRefreshInterval specifies the minimum refresh interval to be used
+		# when using `jwk.Cache`. This value is ONLY used if you did not specify
+		# a user-supplied static refresh interval via `WithRefreshInterval`.
+		#
+		# This value is used as a fallback value when tokens are refreshed.
+		#
+		# When we fetch the key from a remote URL, we first look at the max-age
+		# directive from Cache-Control response header. If this value is present,
+		# we compare the max-age value and the value specified by this option
+		# and take the larger one.
+		#
+		# Next we check for the Expires header, and similarly if the header is
+		# present, we compare it against the value specified by this option,
+		# and take the larger one.
+		#
+		# Finally, if neither of the above headers are present, we use the
+		# value specified by this option as the next refresh timing
+		#
+		# If unspecified, the minimum refresh interval is 1 hour
+		min_refresh_interval 10s
+	}
+}
+
+{
+	# Many directives manipulate the HTTP handler chain and the order in which
+	# those directives are evaluated matters. So the jwtauth directive must be
+	# ordered.
+	# c.f. https://caddyserver.com/docs/caddyfile/directives#directive-order
+	order jwtauth before basicauth
+
+	# Local env dev config
+    admin 0.0.0.0:2019
+	debug
```

Comment:
Can you make this optional?
Other services use the "DEBUG" and "DEV" environment variables (we can set DEBUG in production).